dependencies.props

```diff
@@ -9,17 +9,17 @@
        These ref versions are pulled from https://github.com/dotnet/versions.
   -->
   <PropertyGroup>
-    <CoreFxCurrentRef>bc189e7681f4e64bb2ba779eefb40b9e4542c683</CoreFxCurrentRef>
-    <CoreClrCurrentRef>bc189e7681f4e64bb2ba779eefb40b9e4542c683</CoreClrCurrentRef>
+    <CoreFxCurrentRef>b8a36b6b834f7c5c17824482cce228fefacab563</CoreFxCurrentRef>
+    <CoreClrCurrentRef>b8a36b6b834f7c5c17824482cce228fefacab563</CoreClrCurrentRef>
     <StandardCurrentRef>807c5b1aab825f6c4fde32b333bbe7d6ab59678e</StandardCurrentRef>
     <BuildToolsCurrentRef>252c6a7d142bae7d2cce3e9b837236867b76c1b3</BuildToolsCurrentRef>
   </PropertyGroup>
 
   <PropertyGroup>
-    <MicrosoftNETCorePlatformsPackageVersion>2.1.0-preview3-26430-04</MicrosoftNETCorePlatformsPackageVersion>
+    <MicrosoftNETCorePlatformsPackageVersion>2.2.0-preview3-26501-01</MicrosoftNETCorePlatformsPackageVersion>
     <MicrosoftNETCoreTargetsPackageVersion>2.0.0</MicrosoftNETCoreTargetsPackageVersion>
-    <MicrosoftPrivateCoreFxNETCoreAppPackageVersion>4.5.0-preview3-26430-04</MicrosoftPrivateCoreFxNETCoreAppPackageVersion>
-    <MicrosoftNETCoreRuntimeCoreCLRPackageVersion>2.2.0-preview1-26430-05</MicrosoftNETCoreRuntimeCoreCLRPackageVersion>
+    <MicrosoftPrivateCoreFxNETCoreAppPackageVersion>4.6.0-preview3-26501-01</MicrosoftPrivateCoreFxNETCoreAppPackageVersion>
```

Comment:
@weshaggard - how come corefx is producing `4.6.0-preview3` packages?  Shouldn't this have been bumped back down to `preview1` when we moved to `4.6`?

Reply:
LOL in all my issues with versioning I forget to move it back to preview1 I will do that now.

Reply:
Fixed https://github.com/dotnet/corefx/commit/9abeaf8dc5c7f35036999ccde2b28ef27f72ca7a